libhb/enc_qsv.c

```diff
@@ -315,12 +315,12 @@ static int qsv_hevc_make_header(hb_work_object_t *w, mfxSession session)
     }
 
     /* need more space for 10bits */
-    int bpp12 = 3;
-    if (pv->param.videoParam->mfx.FrameInfo.FourCC == MFX_FOURCC_P010)
+    if (pv->param.videoParam->mfx.FrameInfo.FourCC == MFX_FOURCC_P010 ||
+        pv->param.videoParam->mfx.CodecId == MFX_CODEC_HEVC)
     {
          hb_buffer_realloc(bitstream_buf,bitstream_buf->size*2);
-         bpp12 = 6;
-    }
+	}
```

Comment:
please use spaces instead of tabulation

Reply:
Sorry for that.
I fixed to use spaces.

Reply:
good,
please use commit subject as- **qsv:** ...............

Reply:
OK.
I rebased -i to squash 2 commits and updated the subject that start with `qsv:`.
And it is force pushed.

Reply:
thank you
LGTM